src/raudio.c

```diff
@@ -420,7 +420,7 @@ static bool SaveFileText(const char *fileName, char *text);         // Save text
 // NOTE: Those functions are not exposed by raylib... for the moment
 //----------------------------------------------------------------------------------
 AudioBuffer *LoadAudioBuffer(ma_format format, ma_uint32 channels, ma_uint32 sampleRate, ma_uint32 sizeInFrames, int usage);
-void UnloadAudioBuffer(AudioBuffer *buffer);
+void UnloadAudioBuffer(AudioBuffer *buffer, bool freeSampleData);
```

Comment:
I'd prefer to avoid the extra parameter in the `UnloadAudioBuffer()`. As a custom `UnloadSoundAlias()` is provided, it could implement the custom unloading directly inside it.

Reply:
Understandable. PR updated.